packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -1163,6 +1254,14 @@ class _DefaultSemanticsGestureDelegate extends SemanticsGestureDelegate {
         longPress.onLongPressEnd(const LongPressEndDetails());
       if (longPress.onLongPressUp != null)
         longPress.onLongPressUp();
+      if (longPress.onSecondaryLongPressStart != null)
```

Comment:
I'm not entirely sure that this block is necessary, since I'm not sure what the semantics expects/supports here.

Reply:
I don't think it's expected to call secondary callbacks in the default semantic handler. What kind of behaviors does a secondary long press do currently?

Reply:
It doesn't do anything that I know of.